nginx-log-metrics/README.md

```diff
@@ -0,0 +1,66 @@
+# Nginx log metrics demo
+
+## Quick Start
+
+Build and start with proxy:
+
+```shell
+docker compose build --build-arg https_proxy=http://172.17.0.1:7890
+docker compose up -d --force-recreate
```

Comment:
`--force-recreate` should be optional, I suggest to remove it in case that user doesn't know its meaning.

Reply:
Yes. I'd prefer to remove `-d` also because open a new terminal is easy, but having a background job is more confusing.